### ISSUE TITLE
`rambler reverse --all` returns exit code 0 on failure

### ISSUE THREAD

**Issue by ebachle** [elwinar/rambler#42]
When we reverse a series of migrations and one fails in the middle, rambler returns exit code 0 instead of something non-zero indicating a failure.

Sample of a script we were using that showed this symptom:

```
#!/usr/bin/env sh

set -euo pipefail

rambler reverse --all
20:56 info  found 14 available migrations
20:56 info  found 14 applied migrations
20:56 info  14 migrations to reverse
20:56 info  reversing 00074_create-catalog-user.sql
20:56 info  reversing 00073_create-etls-user.sql
20:56 info  reversing 00072_create-periscope-user.sql
20:56 info  reversing 00071_create-fakebook-user.sql
unable to reverse migration 00071_create-fakebook-user.sql: pq: role "fakebook" cannot be dropped because some objects depend on it

DROP user fakebook;

$ echo $?
0
```

**Comment by ebachle** [comment]
Couple other pieces... I'm seeing this with applies too:

```
rambler reverse --all
21:44 info  applying 00075_create-catalog-user.sql
unable to apply migration 00075_create-catalog-user.sql: pq: syntax error at or near "bad"
bad sql;

echo $?
0
```

Also we're running v4.2.0 built in our container from source with `GO111MODULE=on go get github.com/elwinar/rambler@v4.2.0`

**Comment by elwinar** [comment]
Hi.

I've tried to reproduce your situation with a simple migration, but I don't have the same behaviour (status is correctly `1`).

Could you please give me a minimal working example in the form of a configuration, migration files and script exhibiting the bug?

**Comment by elwinar** [comment]
Nevermind, I've just looked at the changelog, the feature was implemented in v4.2.1. So, you just update your version and you'll be fine.
